org.eclipse.jdt.ls.core/src/org/eclipse/jdt/ls/core/internal/HoverInfoProvider.java

```diff
@@ -156,6 +163,34 @@ public HoverInfoProvider(ITypeRoot aUnit, PreferenceManager preferenceManager) {
 		return res;
 	}
 
+	private String fixSnippet(String value) {
+		if (value.contains(JavaDocSnippetStringEvaluator.SNIPPET)) {
+			StringBuilder builder = new StringBuilder();
+			value.lines().forEach((line) -> {
+				if (line.contains(JavaDocSnippetStringEvaluator.SNIPPET)) {
+					line = line.replaceAll("^\\s+", "");
```

Comment:
Could this be replace by `line.stripLeading()` ?

Reply:
Fixed.